app/controllers/admin/students_controller.rb

```diff
@@ -1,28 +1,45 @@
 class Admin::StudentsController < ApplicationController
+  before_action :set_enrollment, only: [:create]
```

Comment:
Why do we need this?

Reply:
Should it be in new? If yes, because we're not repeating it, it is unnecessarily adding extra lines of code. It also doesn't really need to be privatized.

Reply:
Helper methods are typically private, although you are right that it isn't necessary to DRY up code here (yet).